<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1376</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>邮票分类</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/mainbackground/Logo_of_Beijing_University_of_Posts_and_Communications2.svg</normaloff>:/mainbackground/Logo_of_Beijing_University_of_Posts_and_Communications2.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>59</y>
      <width>1271</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>未选择图片</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{backkground-color:white
}</string>
       </property>
       <property name="text">
        <string>未选择图片</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{backkground-color:red
}</string>
       </property>
       <property name="text">
        <string>无分类结果</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1351</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/QtProject/ui/view.jpg);
background-position:center;
background-repeat: no-repeat</string>
    </property>
    <property name="flow">
     <enum>QListView::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>650</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);
border-radius: 10px; border: 2px groove gray;
border-style: outset</string>
    </property>
    <property name="text">
     <string>选择正面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>650</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);
border-radius: 10px; border: 2px groove gray;
border-style: outset</string>
    </property>
    <property name="text">
     <string>分类</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>365</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>正面图片</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>365</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>背面图片</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListView" name="listView_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>660</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/QtProject/ui/Logo_of_Beijing_University_of_Posts_and_Communications2.svg);
background-position:center;
background-repeat: no-repeat;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>710</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);
border-radius: 10px; border: 2px groove gray;
border-style: outset</string>
    </property>
    <property name="text">
     <string>标记、打分</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>30</y>
      <width>365</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>分类结果</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>710</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);
border-radius: 10px; border: 2px groove gray;
border-style: outset</string>
    </property>
    <property name="text">
     <string>选择背面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>650</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);
border-radius: 10px; border: 2px groove gray;
border-style: outset</string>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>640</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Bahnschrift&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>770</y>
      <width>1071</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>680</y>
      <width>231</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color:rgb(255, 0, 0);
font: 18pt &quot;Arial&quot;;
border-style: outset</string>
       </property>
       <property name="text">
        <string> 打分结果：无</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color:rgb(255, 0, 0);
font: 18pt &quot;Arial&quot;;
border-style: outset</string>
       </property>
       <property name="text">
        <string> 划分等级：无</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>790</y>
      <width>1071</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>710</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);
border-radius: 10px; border: 2px groove gray;
border-style: outset</string>
    </property>
    <property name="text">
     <string>正面标注图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>710</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);
border-radius: 10px; border: 2px groove gray;
border-style: outset</string>
    </property>
    <property name="text">
     <string>背面标注图</string>
    </property>
   </widget>
   <zorder>listView</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>listView_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>progressBar</zorder>
   <zorder>label_9</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>label_10</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
  <include location="imgs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cls_msg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slc_msg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>label_and_score()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>599</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cls_msg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>729</y>
    </hint>
    <hint type="destinationlabel">
     <x>672</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>reset_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1020</x>
     <y>671</y>
    </hint>
    <hint type="destinationlabel">
     <x>1310</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_child()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>895</x>
     <y>732</y>
    </hint>
    <hint type="destinationlabel">
     <x>1312</x>
     <y>701</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_child()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1001</x>
     <y>729</y>
    </hint>
    <hint type="destinationlabel">
     <x>1359</x>
     <y>726</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slc_msg()</slot>
  <slot>cls_msg()</slot>
  <slot>label_and_score()</slot>
  <slot>reset_all()</slot>
  <slot>show_child()</slot>
 </slots>
</ui>
